caches/infinispan/src/test/java/org/commonjava/maven/galley/cache/infinispan/FastLocalCacheProviderTest.java

```diff
@@ -161,7 +161,7 @@ public void testConstructorWitNFSSysPath()
     {
         System.setProperty( FastLocalCacheProvider.NFS_BASE_DIR_KEY, temp.newFolder().getCanonicalPath() );
         new FastLocalCacheProvider( new PartyLineCacheProvider( temp.newFolder(), pathgen, events, decorator ), cache,
-                                    pathgen, events, decorator, executor );
+                                    pathgen, events, decorator, executor, null );
```

Comment:
I guess you'd have to call the NFSCacheOwnerProducer here instead if we eliminate null support in the provider itself.